tests/bluetooth/tester/src/btp_csip.c

```diff
@@ -61,6 +108,11 @@ static void csip_discover_cb(struct bt_conn *conn,
 
 	LOG_DBG("Found %zu sets on member[%u]", set_count, conn_index);
 
+	memcpy(sirk, member->insts[0].info.set_sirk, sizeof(member->insts[0].info.set_sirk));
+	size_t sirk_size = ARRAY_SIZE(sirk);
```

Comment:
Suggest to declare variables at top of scope

Reply:
Done